formwidgets/RevisionHistory.php

```diff
@@ -173,26 +168,70 @@ private function validateInput()
     private function getDiff($fieldName, $oldValue, $newValue)
     {
         $fields = $this->parentForm->getFields();
-
-        if (array_key_exists($fieldName, $fields) && isset($fields[$fieldName]->config['revisions'])) {
+        if (array_key_exists($fieldName, $fields)
+            && isset($fields[$fieldName]->config['revisions'])
+        ) {
             $relationConfig = $fields[$fieldName]->config['revisions'];
             $relationClass = $relationConfig['relation'];
-
             if (method_exists($relationClass, 'withTrashed')) {
                 $oldModel = $relationClass::withTrashed()->find($oldValue);
                 $newModel = $relationClass::withTrashed()->find($newValue);
             } else {
                 $oldModel = $relationClass::find($oldValue);
                 $newModel = $relationClass::find($newValue);
             }
-
             $nameFrom = $relationConfig['nameFrom'] ?? 'name';
             return Diff::htmlDiff(
                 e($oldModel->$nameFrom ?? 'Deleted relation'),
                 e($newModel->$nameFrom ?? 'Deleted relation')
             );
         }
-
         return Diff::htmlDiff(e($oldValue), e($newValue));
     }
+
+    /**
+     * Find revision, check if exist and delete this
+     */
+    public function onDeleteRevisionById()
+    {
+        if ($this->readOnly) {
+            Flash::success(Lang::get('samuell.revisions::lang.revision.read_only_error'));
```

Comment:
Please change this to an error flash message

Reply:
> Sorry, but what do you mean with 'See top comment'?
> What I meant with diff styling is that in the ajax popup, the styling is missing like in the history list:
> ![image](https://user-images.githubusercontent.com/20305359/95957568-29435700-0e00-11eb-9a71-28bd7a03b4be.png)

Oh my mistake. I thought you meant width styles for popup.

Reply:
@rubenvanerk 
Is done. I added class "revision history" to popup partial.